vsts-fullbuild-redhat-vm/README.md

```diff
@@ -0,0 +1,59 @@
+# VM-Redhat - Team Services Build Agent and Cross-Platform SDKs installation
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Fvsts-fullbuild-redhat-vm%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Fvsts-fullbuild-redhat-vm%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template allows you to create an Red Hat VM with a full cross-platform development environment to support:
+* OpenJDK 6, 7 and 8; 
+* Ant, Maven and Gradle;
+* npm and nodeJS;
+* groovy and gulp;
+* Gnu C and C++ along with make;
+* Perl, Python, Ruby and Ruby on Rails;
+* .NET;
+* go; and
+* the VSTS CoreCLR Linux Build Agent. 
+(Build Agent available here: https://github.com/Microsoft/vsts-agent).
+
+To learn more about Visual Studio Team Services (VSTS) and Team Foundation Server (TFS) support for Java, check out:
+http://java.visualstudio.com/
+
+
+## Before you Deploy to Azure
+
+To create the VM, you will need to:
+
+1. Know the Team Services URL (e.g. https://myaccount.visualstudio.com)
+
+2. Create or obtain a Personal Access Token (PAT) from Team Services which has *"Build (read and execute)"* and *"Agent Pools (read, manage)"* priviledges/capabilities
+(see https://www.visualstudio.com/en-us/docs/setup-admin/team-services/use-personal-access-tokens-to-authenticate).
+
+3. Create or obtain a build agent pool in Team Services
+(see https://www.visualstudio.com/en-us/docs/release/getting-started/configure-agents)
+
+4. Decide on a name for your build agent (i.e. the name for your agent within the above pool).
+
+
+## Verifying the Agent
+Once the VM is successfully provisioned, Team Services build agent installation and initialization can be verified by accessing the the *Agent pools* tab under the Control panel for the Team Services account
+(e.g. https://myaccount.visualstudio.com/_admin/_AgentPool).  You should be able to click on the build agent pool (from #3 above)
+and see your agent listed by the name (used in #4 above).  If all is well, the colored bar to the left of the pool name should be green.
+If the colored bar is red, or if the agent name does not appear in the specified pool, see below for debugging hints.
+
+
+## Debugging Agent Failures
```

Comment:
We don't have a doc or web page that specifically talks about this Quickstart ARM template other than this README.  However, I will add a link for more general information on administering builds and build agents...